stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml

```diff
@@ -34,6 +34,9 @@ spec:
             - '--probe.docker.bridge={{ .Values.dockerBridge }}'
             - '--probe.docker=true'
             - '--probe.kubernetes=true'
+            {{ if .Values.disableFrontendControls }}
+            - "-probe.no-controls"
```

Comment:
Double-dash here to match the style of the other arguments, though I wouldn't do a commit just for that if this works.